docs/reference/patch-acceptance-criteria.rst

```diff
@@ -34,7 +34,7 @@ For a detailed description on how to format patches before submission, see this
 `guideline document`_, while here is a quick (and far from complete)
 introduction.
 
-.. _guideline document: https://wiki.ubuntu.com/Kernel/Dev/StablePatchFormat
+.. _guideline document: /reference/stable-patch-format
```

Comment:
When referencing files within the same repo we can use :doc:`/reference/stable-patch-format` instead. If the file is moved or doesn't exist this would cause the build itself to fail instead of relying on link checks.

Reply:
Done. Thanks!